static/app/components/events/interfaces/crashContent/exception/sourcemapsWizard.tsx

```diff
@@ -43,6 +53,7 @@ interface Props {
 
 export default function SourceMapsWizard({analyticsParams}: Props) {
   const isDarkmode = useLegacyStore(ConfigStore).theme === 'dark';
+  const {organization, project_id} = analyticsParams;
```

Comment:
wouldn't it be better if we pass this explicitly instead of grabbing it from the `analyticsParams`?